### PR TITLE
Do not set None to external_symbolizer_path env variable (#1346).

### PR DESCRIPTION
**Pull request by Dor1s**
Quick follow up.

### CODE REVIEW

File: src/python/system/environment.py [hunk 1]
```diff
@@ -774,7 +774,10 @@ def update_symbolizer_options(tool_options, symbolize_inline_frames=False):
   if 'external_symbolizer_path' not in tool_options:
     llvm_symbolizer_path_arg = _quote_value_if_needed(
```
Comment (inferno-chromium) [inline src/python/system/environment.py]: i think _quote_value_if_needed should be moved inside the if.
and then s/llvm_symbolizer_path_arg/llvm_symbolizer_path
we shouldn't be quoting None as then we wont fail on the if.

Comment (Dor1s) [reply src/python/system/environment.py]: Good call, fixing!

**Pull request merged**